sematic/ui/packages/main/src/hooks/logHooks.ts

```diff
@@ -97,15 +129,17 @@ export function useAccumulateLogsUntilEnd(hasMore: boolean, getNext: () => Promi
             }
 
             setIsLoading(true);
-            const {pulledLines} = await latestGetNext.current();
+            const {pulledLines} = await latestGetNext.current(DiagnosticReasons.ACCUMULATE);
 
             if (abortController.signal.aborted) {
                 break;
             }
             setIsLoading(false);
             
-            accumulatedLines += pulledLines;
-            setAccumulatedLines(accumulatedLines);
+            if (!isNaN(pulledLines)) {
```

Comment:
This fixes #690

Reply:
Please add a code comment that the server shouldn't send NaN there, but it does for unknown reasons, so we need to be defensive.

Reply:
Sounds good. Done.